### PR TITLE
Menu bar

### PR DESCRIPTION
**Pull request by AriGood**
everything but pushing now works from the menu bar

### CODE REVIEW

**Comment by dariosanp** [comment]
Approved

**Comment by marburcea** [comment]
Approved

File: src/main/test/java/org/use_case/git/CredentialUseCaseTest.java
Comment (mariana-novoa) [inline src/main/test/java/org/use_case/git/CredentialUseCaseTest.java:1]: Have you achieved 100% Test coverage yet?

**Comment by mariana-novoa** [comment]
Ari this looks really good. Your methods are very clear (Javadoc comments) and you already wrote down your tests. Do you still have any other checkstyle errors?

**Comment by mariana-novoa** [comment]
Approved.

**Comment by AriGood** [comment]
No will add more

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**